test/copy.bats

```diff
@@ -0,0 +1,40 @@
+load helpers
+
+function setup() {
+    stacker_setup
+}
+
+function teardown() {
+    cleanup
+}
+
+@test "built type layers are restored correctly" {
+    cat > stacker.yaml <<EOF
+parent:
+    from:
+        type: oci
+        url: $CENTOS_OCI
+    run: |
+        touch /root/parent
+        cat /proc/self/mountinfo
+child:
+    from:
+        type: built
+        tag: parent
+    copy:
+      - /stacker/root/parent -> /root/parent
```

Comment:
Is there a clean way we can augment this to optionally specify uid, gid, and perms?

Something like

```
copy:
  - /stacker/bar -> /root/bar
  - /stacker/foo -> /usr/bin/foo 0:daemon 0755
  - /stacker/baz -> /etc/baz 0:0 644
  - ```

Reply:
The solution for this would be an intermediate layer which fixes the perms before "copy"ing the file.
We can do this, but a minimal directive preferable?

Reply:
For copying files which were built by some other layer, that makes sense (though it still may be comforting to see the final owner+perms in the final stackerfile, but that's somewhat subjective).  But if I'm shipping a git tree that just contains two files to copy, and a stackfile which just creates an empty oci image with just those two files (for instance, a manifest and manifest signature), from: empty, then you're making me download some sort of distro image just to build an intermediate layer in which to import those two files, chown them, and then let the final layer export them.

So IMO if we're going to support copy, we may as well make it as easy as possible to use...